doc/CMakeLists.txt

```diff
@@ -31,7 +31,8 @@
 
 find_program(PANDOC pandoc)
 if (NOT PANDOC)
-        message(WARNING "pandoc not found")
+        message(FATAL
+	    "pandoc not found, needed for generating manpages -- set WITH_INSTALL_TARGETS to OFF to disable manpage generation")
```

Comment:
Man pages are regenerated on demand, so I don't see why we have to fail the build when pandoc is not available.